SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -124,19 +140,19 @@ bool USpatialActorChannel::IsSingletonEntity()
 	return NetDriver->GlobalStateManager->IsSingletonEntity(EntityId);
 }
 
+#if ENGINE_MINOR_VERSION <= 20
 bool USpatialActorChannel::CleanUp(const bool bForDestroy)
+#else
+bool USpatialActorChannel::CleanUp(const bool bForDestroy, EChannelCloseReason CloseReason)
+#endif
 {
 #if WITH_EDITOR
 	if (NetDriver != nullptr)
 	{
 		const bool bDeleteDynamicEntities = GetDefault<ULevelEditorPlaySettings>()->GetDeleteDynamicEntities();
 
-		UWorld* World = NetDriver->GetWorld();
```

Comment:
Why was this removed?

Reply:
World no longer exists at that point in time when shutting down.

Reply:
What about in 4.20?

Reply:
Was there any follow up on this. This was added in https://github.com/spatialos/UnrealGDK/commit/9b24edd369aac989ce6ac9337b5fb35c76b33b0c which still looks valid.

Reply:
Talked to @improbable-valentyn about this. Seems that this is fine considering since it only happens with the editor.